### PR TITLE
CFY-6007: add parameters for waiting for volume status

### PR DESCRIPTION
**Pull request by isaac-s**

### CODE REVIEW

**Comment by idanmo** [comment]
@isaac-s can you please add descriptions to the new inputs in plugin.yaml?

**Comment by isaac-s** [comment]
@idanmo done, and also fixed some `flake8` errors.

File: neutron_plugin/network.py [hunk 1]
```diff
@@ -107,4 +107,6 @@ def delete(neutron_client, **kwargs):
 @with_neutron_client
 def creation_validation(neutron_client, **kwargs):
     validate_resource(ctx, neutron_client, NETWORK_OPENSTACK_TYPE)
+
+
```
Comment (funkyHat) [inline neutron_plugin/network.py]: A better fix here would be to delete the `0` line which serves no purpose

Comment (isaac-s) [inline neutron_plugin/network.py]: It indeed looked odd, but I was way too paranoid. I'll fix this.

Comment (isaac-s) [inline neutron_plugin/network.py]: Done

File: plugin.yaml [hunk 2]
```diff
@@ -403,6 +403,10 @@ node_types:
               default: {}
             openstack_config:
               default: {}
+            status_attempts:
```
Comment (idanmo) [inline plugin.yaml]: description?

Comment (isaac-s) [inline plugin.yaml]: Added. Also added types.

**Pull request merged**